lsp/leanls.lua

```diff
@@ -0,0 +1,46 @@
+---@brief
+--- https://github.com/leanprover/lean4
+---
+--- Lean installation instructions can be found
+--- [here](https://leanprover-community.github.io/get_started.html#regular-install).
+---
+--- The Lean language server is included in any Lean installation and
+--- does not require any additional packages.
+---
+--- Note: that if you're using [lean.nvim](https://github.com/Julian/lean.nvim),
+--- that plugin fully handles the setup of the Lean language server,
+--- and you shouldn't set up `leanls` both with it and `lspconfig`.
+
+---@type vim.lsp.Config
+return {
+  cmd = { 'lake', 'serve', '--' },
+  filetypes = { 'lean' },
+  root_dir = function(bufnr, on_dir)
+    local fname = vim.api.nvim_buf_get_name(bufnr)
+    fname = vim.fs.normalize(fname)
+    -- check if inside elan stdlib
```

Comment:
i assume this is a typo ?

```suggestion
    -- check if inside lean stdlib
```